FAQ.md

```diff
@@ -85,6 +86,24 @@ Some image `.png` files are grayscale. This indicates that even though these ima
 
 It really depends on what image you're trying to change the colors of, where these colors are specified. Try looking for related files or `.pal` files.
 
+## How do I find absolute ROM offsets?
+When building external tools, it is usually necessary to know where the data you need is stored. 
+
+The first step is to browse the directories of the project and check files that look related to what you're looking for.
```

Comment:
A guide written at this level is probably more appropriate for a [wiki](https://github.com/pret/pokecrystal/wiki/Tutorials) page. You'd also have more immediate control that way, not needing a PR process to get your writing out there.